### DISCUSSION TITLE
"API Content" vs "non-API content"

### DISCUSSION THREAD

**Discussion by psmedia** [brianpetro/obsidian-smart-connections#315] [Q&A]
Brian - what a great tool! I am loving it. Question - I'm just seeking reassurance that the notes sent for embedding to Open-AI are sent under the API, and so are considered API Content as per OpenAI's terms of use. SO that material will not be used by OpenAI for further training. Same question about the chat. This makes a big difference in terms of which vaults I would feel comfortable using the plug-in on.

**Comment by felipemeres** [comment]
Yes, everything is going through the API.
